src/Resources/views/Block/block_feature_media.html.twig

```diff
@@ -12,15 +12,9 @@ file that was distributed with this source code.
 {% extends sonata_block.templates.block_base %}
 
 {% block block %}
-    <div class="sonata-media-block-feature-media-container {{ settings.class }}">
```

Comment:
Those clases make no sense, this block is suposed to be used in the user app so he must define the template and all classes and styles.